components/Form.js

```diff
@@ -152,6 +241,62 @@ export default function Form({
         onValueChange={handleCropTypeChange}
         labelButtonText={checkSelectInput(currentCropType, "crop type")}
       />
+      <Label htmlFor="image">Image</Label>
```

Comment:
This should match the id of the input that the label is for. i.e in this case it could be you want `fileInput`